src/Geometry.cpp

```diff
@@ -528,48 +519,49 @@ int Geometry::findFSRId(LocalCoords* coords) {
   std::size_t fsr_key_hash = key_hash_function(getFSRKey(coords));
 
   /* If FSR has not been encountered, update FSR maps and vectors */
-  if (_FSR_keys_map.find(fsr_key_hash) == _FSR_keys_map.end()){
-
-    /* Get the cell that contains coords */
-    Cell* cell = findCellContainingCoords(curr);
-    
-    /* Get the lock */
-    omp_set_lock(_num_FSRs_lock);
-
-    /* Recheck to see if FSR has been added to maps after getting the lock */
-    if (_FSR_keys_map.find(fsr_key_hash) != _FSR_keys_map.end())
-      fsr_id = _FSR_keys_map.at(fsr_key_hash)._fsr_id;
+  if (!_FSR_keys_map.contains(fsr_key_hash)){
+
+    /* Try to get a clean copy of the fsr_id, adding the FSR data 
+       if necessary where -1 indicates the key was already added */
+    fsr_id = _FSR_keys_map.insert_and_get_count(fsr_key_hash, NULL);
+    if( fsr_id == -1)
+    {
+      fsr_data volatile* fsr;
+      do{
```

Comment:
@samuelshaner and @PrezNattyGibbs make some good points. This is really a relic of the old days when OpenMOC was in its very infancy, and we didn't have a consistent style guide. Now that we do - largely based on Google's - let's try to stick to it. Of course this means that there is still some purging left to do to get everything consistent, but we owe it to ourselves to make these sorts of comments during code review to ensure the consistency of new code.

Also, for the record, as @geogunow noted above, there are 253 instances of unspaced parentheses and brackets ") " throughout the code. But there are 942 instances of spaces between parentheses and brackets ") {". So we have about 26% of our inconsistent codebase needs some housecleaning :-)